bot.js

```diff
@@ -71,6 +71,10 @@ var bot_options = {
     studio_command_uri: process.env.studio_command_uri
 };
 
+const SQLDriver = require('./lib/dbDriver/mysql');
+
+global.projectBot = {};
+projectBot.driver = new SQLDriver();
```

Comment:
I'd recommending changing the name of this property to `projectBot.dbDriver` to make it abundantly clear what it is. Drivers are not just for databases, so making a more descriptive name can help with clarity.

Reply:
makes sense!